spyder/plugins/projects/plugin.py

```diff
@@ -512,7 +502,7 @@ def close_project(self):
             self.set_conf('current_project_path', None)
             self.setup_menu_actions()
 
-            self.sig_project_closed.emit(path)
+            self.sig_project_closed.emit(path, False)

```

Comment:
```suggestion
            self.sig_project_closed.emit(path)
            self.sig_project_closed[bool].emit(True)
```

* The first signal needs to be emitted for things to work as before.
* As you can probably guess for the second signal, the emitted value is irrelevant. The important thing is its signature.